packages/client/src/app/game/views/game-room/game-room.component.ts

```diff
@@ -15,7 +15,7 @@ import * as _ from 'lodash';
 })
 export class GameRoomComponent implements OnInit, OnDestroy {
   private gameData$: Observable<GameData.GameData>;
-  private game: GameData.GameData;
+  game: GameData.GameData;
```

Comment:
ERROR in /home/ubuntu/CesiumFPS/packages/client/src/$$_gendir/app/game/views/game-room/game-room.component.ngfactory.ts (113,31   Property 'game' is private and only accessible within class 'GameRoomComponent'.